libraries/Ticker/Ticker.h

```diff
@@ -34,17 +36,22 @@ class Ticker
 public:
 	Ticker();
 	~Ticker();
-	typedef void (*callback_t)(void);
```

Comment:
It might be better to keep this definition for compatibility. I ran grep on my sketchbook and found a few uses of `Ticker::callback_t`. Perhaps others could be using it as well.